### ISSUE TITLE
Request: Quiet or Silent mode

### ISSUE THREAD

**Issue by mehcode** [sagiegurari/cargo-make#117]
First thanks for the awesome tool. 

I'd love a `-s` or `--silent` option on `cargo make` to silence all output from `cargo make` directly. 

I'd also like a `silent = true` option on each `task`.

Thoughts?

**Comment by sagiegurari** [comment]
is ```--loglevel error``` good enough?

**Comment by mehcode** [comment]
```
❯ cargo make --loglevel error test
+ cd /Users/mehcode/Workspace/...
++ command -v diesel
+ '[' -x /Users/mehcode/.cargo/bin/diesel ']'
+ cd /Users/mehcode/Workspace/...
+ diesel setup
+ cd /Users/mehcode/Workspace/...
+ diesel setup
    Finished dev [unoptimized + debuginfo] target(s) in 0.16s
     Running target/debug/deps/kaching_api-27fddb7ea22864ed

[...]
```

Still logs output of each command. I'm thinking of the equivalent of using make with `@` prefixed everywhere.

**Comment by sagiegurari** [comment]
I'll check it. i was sure it wouldn't be printed

**Comment by sagiegurari** [comment]
just pushed a fix for this in the 0.15.0 branch, so you can test it locally if you wish.
there is still a lot to do in 0.15.0 branch so might take a bit longer to release it.

**Comment by sagiegurari** [comment]
fix for this has been published.
if you need anything more or if it doesn't work reopen the issue (or create a new one).
